lib/routes/account.js

```diff
@@ -946,10 +1013,33 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+
+          var doSigninConfirmation = requestHelper.shouldEnableSigninConfirmation(sessionToken.uid.toString('hex'), config)
+
+          if (doSigninConfirmation) {
+            return db.sessionTokenWithVerificationStatus(sessionToken.id)
```

Comment:
So stepping back, what route is currently using the `sessionTokenWithVerificationStatus` strategy and depending on it rejecting unverified tokens?  If we need both, I suggest something like:

* `sessionTokenWithVerificationStatus` -> looks up the status, but doesn't assert it's any particular value
* `sessionTokenVerified` -> we had this name originally and I think it's a good name for when we are in fact insisting that the the session is verified

But it's not clear to me what routes would use a strict `sessionTokenVerified` strategy.